### PR TITLE
Curriculum announcements

### PR DESCRIPTION
**Pull request by aortizoj15**

### CODE REVIEW

File: stories/components/ProgressCard.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import ProgressCard from '../../components/ProgressCard'
+
+export default {
+  component: ProgressCard,
+  title: 'Components/ProgressCard'
+}
+
+export const Basic: React.FC = () => (
+  <ProgressCard progressCount={0}></ProgressCard>
```
Comment (Wolfy64) [inline stories/components/ProgressCard.stories.tsx]: If you don't have children it's a common practice to write it with one self closing tag `<ProgressCard progressCount={0} />`

File: scss/progressCard.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.progress-card__container {
+  display: flex;
+  border-radius: 8px;
+  box-shadow: 0 4px 10px 0 rgba(0, 0, 0, 0.07);
+  background-color: #5438dc;
```
Comment (Wolfy64) [inline scss/progressCard.scss]: Instead to repeat the color everywhere it make more sense to add a CSS variable:
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

File: pages/curriculum.tsx [hunk 3]
```diff
@@ -40,8 +53,16 @@ const Curriculum: React.FC = () => {
     return (
       <Layout>
         <div className="row mt-4">
-          <div className="col-9">{sortedLessons}</div>
-          <div className="col-3"></div>
+          <div className="col-8">{sortedLessons}</div>
+          <div className="col-4">
+            <ProgressCard progressCount={0}></ProgressCard>
```
Comment (Wolfy64) [inline pages/curriculum.tsx]: If you don't have children it's a common practice to write it with one self closing tag `<ProgressCard progressCount={0} />`

File: components/ProgressCard.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+const ProgressCard: React.FC<Props> = ({ progressCount }) => {
+  return (
+    <div className="progress-card__container card shadow-sm mt-3">
+      <div className="card-body">
+        <div className="progress-card__progress-bar">
+          <span className="font-weight-bold">{progressCount}%</span>
+        </div>
+        <h4 className="progress-card__title mt-3">
```
Comment (Wolfy64) [inline components/ProgressCard.tsx]: For HTML Semantic you should not add a `H4` if you don't have H1...H3. Please Change for a H1 that you will style it as a h4

File: components/AnnouncementCard.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  announcementTwo,
+  announcementThree,
+  announcementFour
+}) => {
+  return (
+    <div className="announcement-card__container card shadow-sm mt-3">
+      <div className="card-body">
+        <h4>General Announcements</h4>
+        <div className="announcement-card__paragraph-container">
+          <h6 className="announcement-card__subheading font-weight-bold mt-4">
```
Comment (Wolfy64) [inline components/AnnouncementCard.tsx]: See comment below for `h4` and `h6`

File: stories/components/ProgressCard.stories.tsx (hunk 1 above)
Comment (aortizoj15) [reply stories/components/ProgressCard.stories.tsx]: Fixed 👍

File: pages/curriculum.tsx (hunk 3 above)
Comment (aortizoj15) [reply pages/curriculum.tsx]: Fixed 👍

File: components/ProgressCard.tsx (hunk 4 above)
Comment (aortizoj15) [reply components/ProgressCard.tsx]: Thanks learned something. Fixed 👍

File: components/AnnouncementCard.tsx (hunk 5 above)
Comment (aortizoj15) [reply components/AnnouncementCard.tsx]: Fixed 👍

**Pull request merged**